examples/teams/team-platform/index.ts

```diff
@@ -1,11 +1,12 @@
 import { ArnPrincipal } from "@aws-cdk/aws-iam";
-import { PlatformTeam } from "..";
+
+import { PlatformTeam } from '../../../lib/teams';
 
 export class TeamPlatform extends PlatformTeam {
     constructor() {
         super({
             name: "platform",
-            users: [new ArnPrincipal("arn:aws:iam::929819487611:user/superadmin")]
+            users: [new ArnPrincipal("arn:aws:iam::115717706081:user/superadmin")]
```

Comment:
Why account change? If it is still a personal account (isengard) then I see no reason to change. One approach is to parameterize the account and make it part of the cdk context (see EC2 cluster provider options). Could be obtained as `anyStructLikeStackOrCluster.node.tryGetContext()`.